CONTRIBUTORS.md

```diff
@@ -148,6 +150,7 @@ Created by running `./build-support/bin/contributors.sh`.
 + Sara Solano
 + Senthil Kumaran
 + Sergey Serebryakov
++ Shuo Li
```

Comment:
@shuoli84 : Is this how you'd like to be represented here?

Reply:
yap. thx for mentioning.